mathml/elements/mfrac.json

```
@@ -72,7 +73,7 @@
             "status": {
               "experimental": false,
               "standard_track": true,
-              "deprecated": false
+              "deprecated": true
```

Looking at the thread on mdn/content, it seems that there is no formal decision to deprecate this yet, so we should probably keep this as `false`:

```suggestion
              "deprecated": false
```

---

Yes, let's just take the firefox change for now.